### PR TITLE
feat: Replace some styles with classes

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Wanted to test the utility of our utility classes so did a cmd+f for `style=` and tried to replace a bunch of things. I'll likely keeping doing this chunk by chunk.

Part of the cleaning of our codebase is to remove styles in favor of standard utility classes. 

## Changes

* Little to no changes other than some color tweaks or border tweaks where we were using non-compliant things

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Entirely locally checking one at a time

### CODE REVIEW

File: frontend/src/lib/components/BillingAlerts.tsx [hunk 1]
```diff
@@ -15,29 +14,17 @@ export function BillingAlerts(): JSX.Element | null {
...
-                        </div>
-                    </div>
-                </Card>
-            )}
-            {alertToShow === BillingAlertType.UsageNearLimit && (
-                <Card>
-                    <div style={{ display: 'flex' }}>
-                        <div style={{ flexGrow: 1, display: 'flex', alignItems: 'center' }}>
-                            <WarningOutlined className="text-warning" style={{ paddingRight: 16 }} />
+            <div className="p-6 mt-6 border rounded-lg flex items-center">
```
Comment (benjackwhite) [inline frontend/src/lib/components/BillingAlerts.tsx:17]: One of the more refactored components. I checked out each style individually by hardcoding the values and it looked good

File: frontend/src/lib/utils.tsx [hunk 2]
```diff
@@ -179,22 +179,6 @@ export function Loading(props: Record<string, any>): JSX.Element {
     )
 }
 
-export function TableRowLoading({
```
Comment (benjackwhite) [inline frontend/src/lib/utils.tsx:182]: This isn't used anywhere as far as I could see

File: frontend/src/scenes/insights/filters/PathCleaningFilter.tsx [hunk 3]
```diff
@@ -37,8 +37,8 @@ export function PathCleaningFilter(): JSX.Element {
             <Row align="middle" justify="space-between" style={{ paddingLeft: 0 }}>
                 <Popup
                     visible={open}
-                    placement={'bottom-end'}
-                    fallbackPlacements={['bottom-start']}
+                    placement={'top-end'}
+                    fallbackPlacements={['top-start']}
                     onClickOutside={() => setOpen(false)}
```
Comment (benjackwhite) [inline frontend/src/scenes/insights/filters/PathCleaningFilter.tsx:42]: This is a low down component so it was weird that we preferred bottom. Top makes more sense

**Review by pauldambra — APPROVED** [review]
🚢 

Good example of a change set that a bot would comment on too

and...

Should we label PRs like this so we can link to them and folk can see what the changes would look like when they work?

**Pull request merged**